### PR TITLE
Creation of reports file

### PR DESCRIPTION
**Pull request by kickb71**
Expected Behavior
To create the dashboard page with charts as per the design.
The calendar icon should be on left.

Current Behavior:
I have assigned to create a bar chart where it displays total number of Application received for the past 7 days.

Changes:
I have created the bar chart as expected and also added date picker function.
The calendar icon is on the left because it cant be done.
Also used the design as a reference to create a bar chart.

!Important!
The graph will not appear because of no Data in DB file.

Issue no #811

Picture Of The Graph
image

Please Note:
The graph is bit small because its zoomed out. To take the whole screen shot

### CODE REVIEW

**Comment by GauravGusain98** [comment]
@kickb71 Please fix the checks

**Comment by kickb71** [comment]
> @kickb71 Please fix the checks

Okay

File: resources/views/hr/recruitment/reports.blade.php [hunk 1]
```diff
@@ -2,25 +2,31 @@
...
         </div>
-        <div class="col-md-12">
-            @include('status', ['errors' => $errors->all()])
+        <div class="card-body">
+          <canvas id="line-chart"></canvas>
         </div>
-        <div class="col-md-12">
-            <h1>Recruitment Reports</h1>
-        </div>
-        <div class="col-md-12">
```
Comment (GauravGusain98) [inline resources/views/hr/recruitment/reports.blade.php:18]: Pleaese correct the indentation. It should be 4 spaces or a tab. Currently, it is only 2 spaces indentation.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments. Also if possible can you please add a screenshot of the chart.

**Comment by GauravGusain98** [comment]
@kuldeep3  If possible can you also check this PR once.

File: resources/views/hr/recruitment/reports.blade.php (hunk 1 above)
Comment (kickb71) [reply resources/views/hr/recruitment/reports.blade.php:18]: OK

**Review by GauravGusain98 — APPROVED** [review]
Looks fine. Approving.

**Pull request merged**